magni_bringup/scripts/launch_core.py

```diff
@@ -5,16 +5,21 @@
 # 3DTOF, etc, if they are installed.
 ###
 
-import sys, os, subprocess, time
-import roslaunch
+import sys, os, subprocess, time, argparse
+import roslaunch, rospkg
 import yaml
 import smbus # used for the hw rev stuff
+import em
 
-conf_path = "/etc/ubiquity/robot.yaml"
+rp = rospkg.RosPack()
+conf_path_1 = "/etc/ubiquity/robot.yaml"
+conf_path_2 = rp.get_path('magni_bringup')+"/param/robot.yaml"
+core_em_path = rp.get_path('magni_bringup')+"/param/core_launch.em"
 
 default_conf = \
```

Comment:
I think we can get rid of default_conf here, can use the file in the package as the default.

Reply:
Ok, i removed the hardcoding part of it, but it still needs to exist since we are element-checking with it in the get_yaml() function. Now it just gets imported from `magni_bringup/config/default_robot.yaml`